html-aam/index.html

```diff
@@ -6428,7 +6505,46 @@ <h4 id="el-select-combobox">`select`<span class="el-context"> (with NO `multiple
+                  <li>If the `select` element does not have an `aria-valuetext` attribute and has not calculated a usable value from its button part, then the accessible value is calculated from the accessible name of the selected `option` element, even if the accessible name of the selected `option` is the empty string.</li>
+                </ol>
+                <!-- provide a link to the ARIA spec when the https://github.com/w3c/aria/pull/2417 PR lands --> 
+
+                <p>
+                  The calculated role of the customizable `select` element's popup picker is a `listbox`, by default.</p>
+                <p>
+                  If authors include invalid elements (according to the the `select` element's content model) as descendants of the `select` element, and those invalid elements result in accessible objects being present as siblings in the accessibility tree to the valid descendent elements of the select, then user agents SHOULD expose the popup picker with the role of `dialog` in the modeless state.</p>
+
+                <p>
+                  In the event invalid elements are present, but are not rendered, then no user agent role repair is necessary.</p>
+                <p>
+                  In the event that invalid elements are present, but are empty, include only whitespace, or are only used as wrapping elements for the valid descendent element's of a select, then user agents SHOULD ignore these elements in the accessibility tree.
+                </p>
+
+                <div class=example>
+                  <p>In the following example the declared `article` elements are invalid per the `select` element's content model. But as each `article` element is empty, contains only whitespace, or contains only valid `option` elements, the `article` element's role can be ignored in the user agent's accessibility tree:</p>
+
+                  <pre><code>
+&lt;select>
+  &lt;button>&lt;selectedcontent>&lt;/selectedcontent>&lt;/button>
+  &lt;article> &lt;/article> <!-- empty / containing only white space so can be ignored -->
+  &lt;article> <!-- only contains option elements, so article role can be ignored -->
+    &lt;option>item 1
+    &lt;option>item 2
+  &lt;/article>
+  &lt;article hidden> foo &lt;/article> <!-- article is in the hidden state, so is ignored -->
+&lt;/select>
+                  </code></pre> 
+                </div>
+
+                <p class=note>Treating the picker as a `dialog` can help inform users that content beyond the expected elements of a `listbox` popup are present, as they can then inspect the content of the dialog with their assistive technology to discover such content.</p>
+
+                <p>See also <a href="#el-button-select-child">`button` element as a child of a `select` element.</p>
```

Comment:
For better readability, should we move (or duplicate) the link to the 'button as a child of a select element' section closer to the second bullet point in the accessible name computation? It might help readers understand, for example, why an aria-label set on the button doesn’t contribute to the accessible name computation.